sequencer/src/block.rs

```diff
@@ -210,6 +232,22 @@ impl BlockHeader for Header {
         block_merkle_tree.push(parent.commit()).unwrap();
         let block_merkle_tree_root = block_merkle_tree.commitment();
 
+        // fetch receipts from the l1
+        let receipts = fetch_fee_receipts();
```

Comment:
Is the ethereum block number the espresso block is tied to known before calling this function?

Reply:
I believe you are referring to `l1_head` field on the `Header`. Which if I understand should be available here as `parent.l1_head`. I may be mistaken though.

Reply:
Yes, although we should use `l1_finalized` instead, so we don't have to deal with L1 reorgs